src/test/java/com/cloudant/tests/ReplicationTest.java

```diff
@@ -103,33 +102,21 @@ public void replication_filteredWithQueryParams() {
     }
 
     @Test
-    public void replicatorDB() throws Exception {
+    public void replicationDB() throws Exception {
```

Comment:
The name is a bit terse and not very descriptive - `replicateDatabase` would be a bit better.